mw-webapp/src/component/table/Table.module.scss

```diff
@@ -6,29 +6,54 @@ $cell-max-width: 600px;
 .table {
   width: 100%;
   height: 100%;
-  border: $table-border;
```

Comment:
Can you please explain how table's border renders if you deleted this property?

Reply:
I use cell borders instead of table border. Using table borders along with "border-collapse: separate" causes issues with borders: 
- strange table corners; 
- table's border is thicker than expected.

![image](https://github.com/tritonJS826/masters-way/assets/8752900/2bcaadd9-ae09-4ae9-bf51-4ceab8abaa95)